dashboard/config/routes.rb

```diff
@@ -407,6 +408,9 @@ module OPS
   get '/dashboardapi/section_assessments/:section_id', to: 'api#section_assessments'
   get '/dashboardapi/section_surveys/:section_id', to: 'api#section_surveys'
   get '/dashboardapi/student_progress/:section_id/:student_id', to: 'api#student_progress'
+  scope 'dashboardapi', module: 'api/v1' do
+    concerns :section_api_routes
+  end
```

Comment:
Colocating this scope with some future cleanup Josh and I have planned - these `/dashboardapi/section_*/<section-id>` routes should become section actions at `/dashboardapi/section/<section-id>/*`.